asmcli/lib/installation_dependencies.sh

```diff
@@ -389,14 +389,17 @@ register_cluster() {
 }
 
 add_cluster_labels(){
-  local LABELS; LABELS="$(get_cluster_labels)";
-  local WANT; WANT="$(mesh_id_label)"
-  local NOTFOUND; NOTFOUND="$(find_missing_strings "${WANT}" "${LABELS}")"
-
   local PROJECT_ID; PROJECT_ID="$(context_get-option "PROJECT_ID")"
   local CLUSTER_NAME; CLUSTER_NAME="$(context_get-option "CLUSTER_NAME")"
   local CLUSTER_LOCATION; CLUSTER_LOCATION="$(context_get-option "CLUSTER_LOCATION")"
 
+  # temp workaround for attached clusters
+  if [[ -z "${CLUSTER_LOCATION}" ]]; then return 0; fi
+
+  local LABELS; LABELS="$(get_cluster_labels)";
```

Comment:
Side note, I've noticed that get_cluster_labels does `gcloud container clusters describe` even for memberships, is it expected or should it use `gcloud container hub memberships describe`?

Reply:
I'll follow up on that one afterwards, but I did confirm this is working for on-prem/bare metal customers for the latest Anthos release. Not sure *why* it works but...